src/components/dev-hub/green-bulleted-list.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { P } from '~components/dev-hub/text';
+import { size } from '~components/dev-hub/theme';
+
+const GreenBullet = styled('ul')`
+    color: ${({ theme }) => theme.colorMap.darkGreen};
+    list-style-type: circle;
+`;
+
+const ListItemWithSpacing = styled('li')`
+    margin-bottom: ${size.default};
+`;
+
+export const DefaultBulletText = styled(P)`
+    color: ${({ theme }) => theme.colorMap.devWhite};
+    margin-bottom: 0;
+`;
+
+const GreenBulletedList = ({
+    bulletTextComponent = DefaultBulletText,
+    children,
+    ...props
+}) => {
+    const BulletText = bulletTextComponent;
```

Comment:
Do we need this line or can we just use the prop?

Reply:
hmm we don't necessarily need this line. I could just use the prop but i would have to start the prop name with a capital letter bc i think all React component names have to start with a capital letter. i notice prop names typically start with a lower case letter, so would we be breaking convention if I started one with a capital letter?

Reply:
Let's have the prop start with a capital letter and use that :)